set.mm

```diff
@@ -30532,14 +30532,16 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
     rabeqf $p |- ( A = B -> { x e. A | ph } = { x e. B | ph } ) $=
       ( wceq cv wcel wa cab crab nfeq eleq2 anbi1d abbid df-rab 3eqtr4g ) CDGZB
       HZCIZAJZBKTDIZAJZBKABCLABDLSUBUDBBCDEFMSUAUCACDTNOPABCQABDQR $.
+  $}
 
+  ${
     rabeqif.3 $e |- A = B $.
```

Comment:
So I guess it's now
rabeqif.3 --> rabeqi.1
and rabeqif --> rabeqi for the conclusion.